packages/core/resources/icons/icons.json

```diff
@@ -2840,5 +2840,26 @@
     "tags": "sidebar, layout, spreadsheet",
     "group": "table",
     "content": "\\e778"
+  },
+  {
+    "name": "Tick circle",
+    "className": "pt-icon-tick-circle",
+    "tags": "tick mark, sign, ok, approved, success",
+    "group": "action",
+    "content": "\\e779"
+  },
+  {
+    "name": "Prescription",
```

Comment:
Agree, though there already are a bunch of specific icons that got in previously (i.e. rig, field, known vehicle, blocked person...). We could clean it up in 2.0 by keep specific ones internally?